ambry-protocol/src/main/java/com.github.ambry.protocol/AdminRequest.java

```diff
@@ -70,7 +66,7 @@ public static AdminRequest readFrom(DataInputStream stream, ClusterMap clusterMa
    * @param clientId the ID of the client.
    */
   public AdminRequest(AdminRequestOrResponseType type, PartitionId partitionId, int correlationId, String clientId) {
-    super(RequestOrResponseType.AdminRequest, ADMIN_REQUEST_VERSION_V1, correlationId, clientId);
+    super(RequestOrResponseType.AdminRequest, ADMIN_REQUEST_VERSION_V2, correlationId, clientId);
```

Comment:
I thought about it but it seemed like an unnecessary extra variable. In terms of making a change, this is a minor refactoring (and also package-private, so scope is limited).